tests/lac/solver_control_06.expect=run.debug.output

```diff
@@ -0,0 +1,2 @@
+
+content irrelevant
```

Comment:
because the test is expected to fail at run-time, that's decoded in it's name `solver_control_06.expect=run.debug.output`, `expct=run` means that it runs but does not exit gracefully.